changelog.md

```diff
@@ -1,3 +1,105 @@
+<a name="6.1.0"></a>
+# 6.1.0 (2020-06-25)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.0.0...v6.1.0)
+
+We expect this release to ship in the DevTools of [Chrome 85](https://chromiumdash.appspot.com/schedule).
+
+## New Contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+Loftie Ellis <lpellis@gmail.com>
+Marvin Frachet <marvin.frachet@gmail.com>
+Matt Hobbs <matthew.hobbs@digital.cabinet-office.gov.uk>
+Peter Marshall <p.s.marshall0@gmail.com>
+Umar Hansa <umar.hansa@gmail.com>
```

Comment:
we should really try to map these back to github login, I guess we could replace it with a github API lookup on the commit itself?

I can look into that

for this release just manual?

```suggestion
Loftie Ellis @lpellis
Marvin Frachet @mfrachet 
Matt Hobbs @Nooshu 
Peter Marshall @psmarshall 
```

Reply:
yes please an ty so much for doing this bit for me :)

ref #9857